### PR TITLE
docs: Improving the Getting Started section and adjacent pages

### PR DESCRIPTION
**Pull request by JohannesDienst-askui**
* Remove User Portal Pages as they were for User Portal 1.0
* Rework _Getting Started_ page:
** Consistent use of terms
** Reworked structure to reflect the new onboarding workflow
* Refactor the page _Tooling/annotation_  to explain interactive annotation in a concise as visual way
* Adapt the **example_projects_template_** to match the _Getting Started_ docs

@adi-wan-askui Will the changes to the **example_projects_template_** be deployed immediately to our customers when we merge this change, or do we have to build a new version for it? I ask because I have to reshoot some parts of the Onboarding videos to match the changed files 😉

### CODE REVIEW

File: docs/docs/general/02-Getting Started/write-your-first-instruction.md [hunk 1]
```diff
@@ -51,74 +48,130 @@ Then, go to your `helper/jest.setup.ts` and add the configuration for your `work
...
+  it('should click on my element', async () => {
+    await aui
+      .click()
+      .text().withText("node_nodu")
+      .exec();
+  });
+});
+````
+
+As before, run your code with `npx jest --config ./test/jest.config.ts`
```
Comment (mlikasam-askui) [inline docs/docs/general/02-Getting Started/write-your-first-instruction.md:126]: not same code , 
the prev code was `npx jest test/my-first-askui-test-suite.test.ts --config ./test/jest.config.ts `

File: docs/docs/general/02-Getting Started/write-your-first-instruction.md [hunk 2]
```diff
@@ -51,74 +48,130 @@ Then, go to your `helper/jest.setup.ts` and add the configuration for your `work
...
+* Read [Selecting an Element by Visual Relation](../03-Best%20Practice/selecting-by-visual-relation.md) to understand the *Relational Selectors* in askui.
+* Read [Selecting an Element with Text](../03-Best%20Practice/selecting-with-text.md) to learn about *Text Selectors* in askui.
+* Read [Speed Up Execution](../03-Best%20Practice/speed_up_execution.md) to understand more about the inference's performance.
+
+### Annotation
+* Read [Debug with Annotation](../05-Tooling/annotation.md) to learn how to use the **Image Annotation Feature**.
+
+### Continuous Integration
+- Read the [askui UI Controller Docker Images](../04-Continuous%20Integration/askui-ui-controller-docker-images.md) to learn more about running askui inside a Docker container.
+- Read the [Gitlab CI/CD](../04-Continuous%20Integration/gitlab-ci.md) to learn about integrating askui into your Gitlab CI/CD.
```
Comment (mlikasam-askui) [inline docs/docs/general/02-Getting Started/write-your-first-instruction.md:177]: for the new images, 
we don't support colors anymore. you can use the latest askui version to get results without colors

File: docs/docs/general/03-Best Practice/selecting-by-visual-relation.md [hunk 3]
```diff
@@ -1,12 +1,22 @@
...
+A common problem while writing instructions which one encounters is interacting with elements which share the same text.
 
-Relational selectors can be difficult and sometimes confusing to work with at first. Sometimes it is even not clear why a specific element will not be selected or why the selector does not seem to work.
+An example can be seen when you interactively annotate your Visual Studio Code Editor.
+Notice that the magnifying glass icon and many other elements throughout the UI share the same name:
 
-After reading this tutorial you will know how to use the full power of all the relational selectors. Additionally, you learn what pitfalls you can fall into and how to avoid them in the future.
+![Icon mobile annotation one.](images/relational_selectors_same_icon1.png)
+![Icon mobile annotation one.](images/relational_selectors_same_icon2.png)
+![Icon mobile annotation one.](images/relational_selectors_same_icon3.png)
```
Comment (mlikasam-askui) [inline docs/docs/general/03-Best Practice/selecting-by-visual-relation.md:9]: for the new images,
we don't support colors anymore. you can use the latest askui version to get results without colors

File: packages/askui-nodejs/example_projects_templates/typescript_jest/test/my-first-askui-test-suite.test.ts [hunk 4]
```diff
@@ -1,15 +1,14 @@
...
 describe('jest with askui', () => {
-  it('should click on text', async () => {
-    // Run this to see what askui annotates
+
+  it('should generate an interactive annotation', async () => {
     await aui.annotateInteractively();
+  });
 
-    await aui.moveMouse(0, 0).exec();
+  it('should click on my element', async () => {
```
Comment (mlikasam-askui) [inline packages/askui-nodejs/example_projects_templates/typescript_jest/test/my-first-askui-test-suite.test.ts:9]: should we disable this and let the user enable it after replacing the comment?

**Comment by mlikasam-askui** [comment]
> * Remove User Portal Pages as they were for User Portal 1.0
> 
>     * Rework _Getting Started_ page:
>       ** Consistent use of terms
>       ** Reworked structure to reflect the new onboarding workflow
> 
>     * Refactor the page _Tooling/annotation_  to explain interactive annotation in a concise as visual way
> 
>     * Adapt the **example_projects_template_** to match the _Getting Started_ docs
> 
> 
> @adi-wan-askui Will the changes to the **example_projects_template_** be deployed immediately to our customers when we merge this change, or do we have to build a new version for it? I ask because I have to reshoot some parts of the Onboarding videos to match the changed files wink

In answer to your question, we have to release a new version in order to deploy the changes inside the example project

File: packages/askui-nodejs/example_projects_templates/typescript_jest/test/my-first-askui-test-suite.test.ts (hunk 4 above)
Comment (JohannesDienst-askui) [reply packages/askui-nodejs/example_projects_templates/typescript_jest/test/my-first-askui-test-suite.test.ts:9]: I thought about it and decided against it as not everybody will be familiar with the concept of using `xit` to ignore a test case.

**Review by mlikasam-askui — APPROVED** [review]

**Pull request merged**